scripts/wsrep_sst_clone.sh

```diff
@@ -0,0 +1,1251 @@
+
+
+    if [ $WSREP_SST_OPT_BYPASS -eq 0 ]
+    then
+        #
+        #  Prepare DONOR for cloning
+        #
+
+        # Use script file to avoid sensitive information on the command line
+        CLONE_EXECUTE_SQL=$(wsrep_mktemp_in_dir "$WSREP_SST_OPT_DATA" --suffix=.sql clone_execute_XXXX)
+        CLONE_PREPARE_SQL=$(wsrep_mktemp_in_dir "$WSREP_SST_OPT_DATA" --suffix=.sql clone_prepare_XXXX)
+
+        cleanup_donor # Remove potentially existing clone user
+
+        export MYSQL_PWD=$ADMIN_PSWD
+        wsrep_log_debug "-> PREPARE DONOR"
+        setup_clone_plugin "donor"
+
+#        wsrep_log_info "REQUIRE_SSL=$REQUIRE_SSL, CLIENT_SSL_OPTIONS=$CLIENT_SSL_OPTIONS"
+        wsrep_log_debug "-> PREPARE DONE"
+
+cat << EOF > "$CLONE_PREPARE_SQL"
+SET wsrep_on=OFF;
+CREATE USER "$WSREP_SST_OPT_REMOTE_JOINER_USER"@"%" IDENTIFIED BY '$WSREP_SST_OPT_REMOTE_JOINER_PSWD' $REQUIRE_SSL;
+GRANT BACKUP_ADMIN ON *.* TO "$WSREP_SST_OPT_REMOTE_JOINER_USER"@"%";
+GRANT SELECT ON performance_schema.* TO "$WSREP_SST_OPT_REMOTE_JOINER_USER"@"%";
+GRANT EXECUTE ON *.* TO "$WSREP_SST_OPT_REMOTE_JOINER_USER"@"%";
+EOF
+        RC=0
+        export MYSQL_PWD=$ADMIN_PSWD
+#        wsrep_log_debug "-> connecting to donor: $CLONE_PREPARE_SQL $MYSQL_PWD $MYSQL_ACLIENT"
+        $MYSQL_ACLIENT --connect-timeout=60 < $CLONE_PREPARE_SQL || RC=$?
+        wsrep_log_debug "-> $RC connect to donor done"
+
+        if [ "$RC" -ne 0 ]
+        then
+            wsrep_log_error "Donor prepare returned code $RC"
+            cat $CLONE_PREPARE_SQL >> /dev/stderr
+            exit $RC
+        fi
+        # Before waiting for the Joiner Clone mysql we send out the message this is a SST
+        wsrep_log_debug "-> NETCAT signal to nc -w 1 $SST_HOST_STRIPPED $WSREP_SST_OPT_REMOTE_HOSTPORT"
+        wsrep_log_debug "-> Signal for CLONE Plugin CLEANUP_CLONE_PLUGIN=$CLEANUP_CLONE_PLUGIN"
+        echo "$CLEANUP_CLONE_PLUGIN|SST@$WSREP_SST_OPT_GTID<EOF>" | nc -w 1 $SST_HOST_STRIPPED $WSREP_SST_OPT_REMOTE_HOSTPORT || :
+
+        # We stay on hold now, waiting for the Joiner to expose the service
+        wsrep_log_info "-> WAIT for Joiner MySQL to be available nc -w 1 -i 1 $SST_HOST_STRIPPED $WSREP_SST_OPT_REMOTE_HOSTPORT"
+        while [ true ]; do
+        	NCPING=`nc -w 1 -i 1 $SST_HOST_STRIPPED $WSREP_SST_OPT_REMOTE_HOSTPORT 2> /dev/null` || :
+            if [ "$NCPING" == "" ]; then
+                wsrep_log_info "[second(s) to timeout: $DONOR_TIMEOUT_WAIT_JOINER_CLONE_INSTANCE]"
+            else
+                wsrep_log_info "Joiner instance available at $SST_HOST_STRIPPED:$WSREP_SST_OPT_REMOTE_HOSTPORT"
+                break
+            fi
+            if [ "$DONOR_TIMEOUT_WAIT_JOINER_CLONE_INSTANCE" == "0" ]; then
+                    wsrep_log_info "TIMEOUT Waiting for the Joiner to setup MySQL clone instance"
+                    wsrep_log_error "TIMEOUT Waiting for the Joiner to setup MySQL clone instance $LINENO"
```

Comment:
There is no need to log the same message once as info and once as an error.

Reply:
removed info